### PR TITLE
Fix READMEs (DOCS-6716)

### PR DESCRIPTION
**Pull request by rmoff**
This is the first step in fixing all of the README files. I've picked one and want to make sure the process is valid for updating them before proceeding with the others. 

/cc @elismaga please can you review, and let me know if this is an appropriate way to do it. I've committed the existing README in Markdown form for https://hub.docker.com/r/confluentinc/cp-kafka-connect and then subsequent edits to it. 

I noticed on DockerHub that it says that READMEs can be set automatically on build - is this something that our tooling would support?

### CODE REVIEW

**Comment by elismaga** [comment]
@rmoff can you send me link to the doc where it said we can set this as part of the image build process? We can definitely try to  incorporate this into our build process.

**Review by elismaga — APPROVED** [review]

**Comment by rmoff** [comment]
@elismaga it's referenced in the hover-over text in the web UI, and corroborated in this SO post: https://stackoverflow.com/a/29135136/350613

**Comment by elismaga** [comment]
> @elismaga it's referenced in the hover-over text in the web UI, and corroborated in this SO post: https://stackoverflow.com/a/29135136/350613

That only works if you use docker hubs build process, which we don't. We would have to add a step to our build scripts to upload the readme when we promote our images to docker hub. I created a ticket to for our team to do that

**Comment by rmoff** [comment]
I've updated the READMEs and I think they're ready for merging and then manual deploy to Docker Hub. 

@ybyzek tagging you in case you or one of your team want to take a quick look over them too :)

File: ce-kafka/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Confluent Enterprise Docker Image for Apache Kafka [Deprecated]
+
+**Whilst still published, this image is deprecated in favor of [cp-server](https://hub.docker.com/r/confluentinc/cp-server)**
```
Comment (ybyzek) [inline ce-kafka/README.md:3]:
```suggestion
**This image is deprecated in favor of [cp-server](https://hub.docker.com/r/confluentinc/cp-server)**
```

File: server-connect-base/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Confluent Server Docker Image for Kafka Connect
+
+Docker image for deploying and running Kafka Connect. 
+
+_Functionally, the `cp-server-connect` and the `cp-server-connect-base` images are identical. Prior to Confluent Platform 6.0 the `cp-server-connect` image included several connectors pre-installed, but **this is no longer the case**._
```
Comment (ybyzek) [inline server-connect-base/README.md:5]: Some are still pre-installed, I think Replicator and maybe FileStream?  Please check me on this.

**Review by ybyzek — APPROVED** [review]
@rmoff I didn't do a thorough technical review (I defer to the experts) but I skimmed and overall LGTM

File: server-connect-base/README.md (hunk 2 above)
Comment (rmoff) [reply server-connect-base/README.md:5]: FileStream is part of Apache Kafka so is included. Replicator is not included.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,13 @@
-# Docker images for Kafka
+# Docker images for Apache Kafka
```
Comment (elismaga) [inline README.md:1]: @rmoff fyi, this file doesn't get uploaded to docker hub. just want to make sure you realize that.

**Review by elismaga — APPROVED** [review]

**Pull request merged**